relay.go

```diff
@@ -50,6 +51,8 @@ type RelayCmd struct {
 
 	GenesisValidatorsRoot string `ask:"--genesis-validators-root" help:"Root of genesis validators"`
 
+	SecretKey string `ask:"--private-key" help:"The relay's secret key used to sign payloads"`
```

Comment:
```suggestion
	SecretKey string `ask:"--secret-key" help:"The relay's secret key used to sign payloads"`
```
Seems better to just use "secret key" since it is used throughout the repo.

Reply:
Yep, forgot that, thanks !